api-tests/src/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { GraphQLClient, gql } from 'graphql-request'
+import { TokenCreateMutation } from '../generated/graphql'
+
+export const baseUrl = import.meta.env.apiEndpoint
+
+export const makeClient = (): GraphQLClient => new GraphQLClient(baseUrl)
+
+export const makeAuthorizedClient = async (): Promise<GraphQLClient> => {
+  const client = makeClient()
+  const mutation = gql`
+    mutation TokenCreate($email: String!, $password: String!) {
+      tokenCreate(email: $email, password: $password) {
+        csrfToken
+        refreshToken
+        token
+        errors: accountErrors {
+          ...AccountError
+        }
+        user {
+          id
+          email
+        }
+      }
+    }
+    fragment AccountError on AccountError {
+      code
+      field
+      message
+    }
+  `
+  const result = await client.request<TokenCreateMutation>(mutation, {
+    email: 'testers+dashboard@saleor.io',
+    password: 'test1234',
```

Comment:
Maybe credentials should also be moved to env variables?

Reply:
https://github.com/saleor/saleor/pull/12963/commits/4c32afa1b18d91352509186d1d0c398f546069f9